lib/cisco_node_utils/command_reference.rb

```diff
@@ -150,7 +180,53 @@ def self.debug=(value)
       @@debug = value # rubocop:disable Style/ClassVars
     end
 
-    attr_reader :files, :product_id
+    KNOWN_APIS = %w(nxapi grpc)
+
+    def self.hash_merge(input_hash, api, product_id, base_hash=nil)
```

Comment:
I think this method could use some documentation.  Lot happening here that is not completely obvious.